Jenkinsfile

```diff
@@ -76,3 +77,18 @@ pipeline {
     }
   }
 }
+
+def waitUntilDbIsReady(def containerId) {
+  timeout(5) {
+    waitUntil {
+       script {
+         try {
+          sh "docker exec ${containerId} mysqladmin -u update -p1234 ping --host=localhost --port=3306 --silent"
```

Comment:
What is this doing?!?

Reply:
It pings the db server. If the db server accepts connection everything works fine. If it's fails then the db server is not ready.

Reply:
I was confused about the `-u update` Is this needed?

Reply:
Yes. I need to pass the user and password.